gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst

```diff
@@ -63,22 +67,24 @@ Now that we have converted several filters to C++ functions, we are ready to cal
    .. code-block:: bash
 
             cd ~/python-pcl_ws/src/
-            catkin create pkg filter_call rospy roscpp 
+            catkin create pkg test_pkg_python --catkin-deps rospy
 
 #. Check that your package was created:
 
    .. code-block:: bash
 
-            ls
+            ls 
 
 We will not be including ‘perception_msgs’ as a dependency as we will not be creating custom messages in this course. If you wish for a more in depth explanation including how to implement customer messages, here is a good `MIT resource <http://duckietown.mit.edu/media/pdfs/1rpRisFoCYUm0XT78j-nAYidlh-cDtLCdEbIaBCnx9ew.pdf>`__ on the steps taken.
 
 
-#. Open *CMakeLists.txt*. You can open the file in Pycharm or Qt (or you can use nano, emacs, vim, or sublime). Uncomment line 23, and save.
+#. Open *CMakeLists.txt*. You can open the file in OT Creator (preferred) or PyCharm (or you can use nano, emacs, vim, or sublime). Uncomment line 19 or wherever you find "# catkin_python_setup() " and save.
```

Comment:
Let's change the verbiage to something like "open the file in a text editor or IDE. Uncomment the line '# catkin_python_setup()` and save. I don't think there's any need to plug specific IDEs